src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -235,6 +235,8 @@ public void spotCheck() throws IOException {
 
             final ArgumentsBuilder arguments = new ArgumentsBuilder();
 
+            arguments.addArgument("verbosity", "INFO");
+
             arguments.addVCF(new File("/Users/jonn/Development/M2_01115161-TA1-filtered.vcf"));
```

Comment:
Please remove all references to `/Users/jonn/Development/` etc

Reply:
This test is intermediate and planned to be phased out.

It is checked in so that I can run my local test data and not have to worry about rewriting tests every time I need to debug something.

I added the flag doDebugTests to control for this and a unit test to ensure that doDebugTests is always set to false for the tests to pass.

I want to leave these in for now so I can continue to test locally with different settings easily.

Reply:
But why can't you put these tests into the source tree and then remove if/when no longer needed?
You are asking to put these into master by issuing this PR and we cannot cut a release with these tests -- and master needs to always be release-ready.  

There should be an alternative way of doing this.